apps/condo/domains/ai/constants.js

```diff
@@ -108,6 +110,28 @@ const FLOW_META_SCHEMAS = {
             },
         },
     },
+    [NEWS_REWRITE_TEXT_FLOW_TYPE]: {
+        input: {
+            type: 'object',
+            properties: {
+                promptType: { enum: ['header', 'body'] },
+                newsTitle: { type: 'string' },
+                newsBody: { type: 'string' },
+
+                newsType: { type: 'string' },
+                validBefore: { type: 'string' },
+                incidents: { type: 'string' },
+                lastNewsTitle: { type: 'string' },
+                lastNewsBody: { type: 'string' },
```

Comment:
Hey, let's revisit these params as:

1. We agreed not to use lastNewsBody / lastNewsTitle / incidents

2. maybe simply title, body, type, validBefore?